### ISSUE TITLE
CVE-2023-2253 - DoS from malicious API request

### ISSUE THREAD

**Issue by pdellamore** [kubernetes/kubernetes#119227]
### What happened?

Trivy image scanners is acusing kubectl for being vulnerable to a CVE that uses a package named as `github.com/docker/distribution`. 

![image](https://github.com/kubernetes/kubernetes/assets/136379529/06b08443-7863-46e2-88ad-945c289ca7db)

### What did you expect to happen?

At the latest kubernetes version (v1.27.3), I have built the image of kubernetes and scanned the image and still have the issue. 

### How can we reproduce it (as minimally and precisely as possible)?

Download trivy scan and run against kubernetes latest image.

### Anything else we need to know?

I would like to know if you guys are addressing this CVE, or can confirm if kubectl binary is really vulnerable to that.

### Kubernetes version

v1.27.3

### Cloud provider

N/A

### OS version

<details>

```console
# On Linux:
$ cat /etc/os-release
# paste output here
$ uname -a
# paste output here

# On Windows:
C:\> wmic os get Caption, Version, BuildNumber, OSArchitecture
# paste output here
```

</details>

### Install tools

<details>

</details>

### Container runtime (CRI) and version (if applicable)

<details>

</details>

### Related plugins (CNI, CSI, ...) and versions (if applicable)

<details>

</details>

**Comment by shivanshu1333** [comment]
Is this needed?
/assign

**Comment by instantlinux** [comment]
The 1.28 version is scheduled for 15-Aug-2023, still a week out. Shouldn't the fix be identified and backported to a 1.27 patch version, and made available for those whose CI pipelines are now blocked?

**Comment by rmontag-ap** [comment]
Can someone confirm that this is fixed in the 1.28.0?

**Comment by mrmadira** [comment]
I am another consumer of kubectl;
I can confirm from my scans that I don't see  CVE-2023-2253 in v28.1
